pages/view/_iscnId.vue

```diff
@@ -303,15 +303,7 @@
             :label="$t('iscn.meta.usage.info')"
             class="mb-[12px]"
           >
-            <Link
-              :class="[
-                'text-[14px]',
-                'break-all',
-              ]"
-              :href="metadata.usageInfo"
-            >
-              {{ metadata.usageInfo }}
-            </Link>
+            {{ metadata.usageInfo }}
```

Comment:
usageInfo can be a url, we should check if it is an url and use link tag, otherwise this patch makes sense